components/components.css

```diff
@@ -0,0 +1,400 @@
+@import "helpers.css";
+
+/* RESETS */
+
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Inter", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin: 0;
+  padding: 0;
+  color: var(--gray-one);
+}
+
+a {
+  text-decoration: none;
+}
+
+:root {
+  --gray-one: #202124;
+  --gray-two: #2f3135;
+  --gray-three: #3c4043;
+  --gray-four: #5f6368;
+  --primary-color: #8b5cf6;
+  /*States Colors*/
+  --success-green: #10b981;
+  --danger-red: #ef4444;
+  --warning-yellow: #fbbf24;
+  --info-blue: #3b82f6;
+}
+
+body {
+  margin: 0;
+}
+
+/* IMAGES */
+
+.img-res {
```

Comment:
Setting image's width and height both to 100% would make it take width and height of the div wouldn't it be hard to maintain the original aspect ratio of the image? Suggestion: either set width:100% or height:100% to maintain the original aspect ratio of the image

Reply:
Thanks for pointing it out, I've made  the changes